lib/stoneacre_hq/prospect.rb

```diff
@@ -8,7 +8,12 @@ def initialize
       @conn = Faraday.new(
         :url => (StoneacreHq.config[:production] ? PRODUCTION_URL : DEVELOPMENT_URL ),
         :ssl => { :verify => StoneacreHq.config[:production] }
-      ) 
+      ) do |builder|
```

Comment:
Yes! I have thought of creating a StoneacreHq::Base class which had this ```initializer``` method, but since it was the only method I didn't bother doing it haha. Lets have that into account when we create more methods.